zson/ztests/mixed-array.yaml

```diff
@@ -1,24 +1,17 @@
-script: |
-  zq -f zson -i zson -pretty=0 in.zson
+zed: "*"
 
-inputs:
-  - name: in.zson
-    data: |
-      [1((int64,string)),"b"((int64,string)),2((int64,string))]
-      [8((int64,string)),3((int64,string))]
-      ["hello"((int64,string)),"goodbye"((int64,string))]
-      [null,1,"hello"]
```

Comment:
Why not keep this?

Reply:
You mean why did I make this a zed test not a script test? I like doing zed tests when I can since they're faster run... is there some direction we're headed w.r.t to when something is a zed test vs. script test?

Reply:
No, I'm referring to the `[null,1,"hello"]` case, which has disappeared.

Reply:
Oh I removed it because there's a test in mixed-null-array that does exactly the same thing. Honestly those two files should probably just be merged into array.yaml....